demo/components/dynamic-content/dynamic-content-details.component.ts

```diff
@@ -77,6 +104,9 @@ export class DynamicContentDetailsComponent implements OnInit {
       this.router.navigate(['/']);
     }
     this.content = article.fields;
+    if (this.homePage) {
+      this.content.relatedLocation = true;
```

Comment:
const contentAtTheBottom = true;
this.content.relatedLocation = contentAtTheBottom;